lib/src/utils/async/simple_stream.dart

```diff
@@ -403,3 +409,27 @@ class SimpleStreamSubscription<T> implements StreamSubscription<T> {
     throw new UnsupportedError('Not supported.');
   }
 }
+
+/// Provides an interface for both StreamController & and Stream for use with
+/// output events in Angular components.
+///
+/// Reduces the amount of boilerplate needed by removing the need for a getter
+/// that returns the stream for angular.
+class SimpleEmitter<T> extends SimpleStream<T> {
+  SimpleEmitter(
+      {bool isSync = true,
+      bool runInZone = true,
+      SubscriptionChangeListener onListen,
+      SubscriptionChangeListener onCancel})
+      : super.broadcast(
+            isSync: isSync,
+            runInZone: runInZone,
+            onListen: onListen,
+            onCancel: onCancel);
+
+  /// Returns referene to object.
```

Comment:
There is a spelling error. How about:

> Returns `this`.

"Returns reference to object" doesn't really tell us which object.